design/ceph/object-store.md

```diff
@@ -43,13 +43,10 @@ kubectl create -f object-store.yaml
 
 At this point the Rook operator recognizes that a new object store resource needs to be configured. The operator will create all of the resources to start the object store.
 1. Metadata pools are created (`.rgw.root`, `my-store.rgw.control`, `my-store.rgw.meta`, `my-store.rgw.log`, `my-store.rgw.buckets.index`)
-1. The data pool is created (`my-store.rgw.buckets.data`)
-1. A Ceph realm is created
-1. A Ceph zone group is created in the new realm
-1. A Ceph zone is created in the new zone group
-1. A cephx key is created for the rgw daemon
-1. A Kubernetes service is created to provide load balancing for the RGW pod(s)
-1. A Kubernetes deployment is created to start the RGW pod(s) with the settings for the new zone
+2. The data pool is created (`my-store.rgw.buckets.data`)
+3. A Kubernetes service is created to provide load balancing for the RGW pod(s)
+4. A Kubernetes deployment is created to start the RGW pod(s) with the settings for the new zone
+5. The zone is modified to add the RGW pod endpoint(s) if realm/zonegroup/zone are mentioned in the configuration
```

Comment:
the zone resource or zone definition in ceph?

Reply:
zone in ceph.